scripts/readme-upload-missing-images.ts

```diff
@@ -0,0 +1,104 @@
+import * as fsPromises from "fs/promises";
+import * as fs from "fs";
+import path from "path";
+import axios from "axios";
+import FormData from "form-data";
+
+const readmeUploadMissingImages = async () => {
+  if (process.env.README_IO_AUTH?.length < 10) {
```

Comment:
let's also consider adding more error handling - try - catch

Reply:
why? I don't want to continue on error.

Reply:
ok, I think it's enough
if something goes wrong with this script, I will get this information in the console, right ?

Reply:
yes